1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
```

Comment:
"# What is the HSR?" would be a better fit. :)

Reply:
Thank you, capitalized noun always confuse me... :)